custom/ewsghana/templates/ewsghana/partials/ews_multibar_chart.html

```diff
@@ -26,7 +26,7 @@
         // Customize tooltip message
         {% if chart.tooltipFormat %}
             multibar_chart.tooltipContent(function(key, y, e, graph) {
-                return '<h3>' + key + '</h3><p>' + e + '{{ chart.tooltipFormat }}' + graph.point.name + ' (' + y + ')';
+                return '<h3>' + graph.point.name + ' (' + y + ')' + '</h3><p>' + e + ' - ' + key;
```

Comment:
it looks like this was already true in the old version, but does this need a `</p>`